src/steamship/data/tags/tag.py

```diff
@@ -303,23 +291,6 @@ def __init__(
         )
 
 
-class EmbeddingTag(Tag):
```

Comment:
Curious why you're deleting this one -- it's super useful. I'll roll forward the various embedder plugins to use it instead of manually creating embedding tags.